include/rdma/fi_log.h

```diff
@@ -35,8 +35,6 @@
 #ifndef FI_LOG_H
 #define FI_LOG_H
 
-#include "config.h"
```

Comment:
user's configuration may be generated not by autotools, and ENABLE_DEBUG may be omit there (and config.h as well), in this case compiltion will be aborted on user's side